langlib/lang.stream/src/main/ballerina/stream.bal

```diff
@@ -148,6 +248,12 @@ public isolated function iterator(stream<Type,CompletionType> stm) returns objec
 
 # Closes a stream.
 #
+# ```ballerina
+# string[] greetings = ["Hello", "Bonjour", "Hola", "Ciao"];
+# stream<string> greetingStream = greetings.toStream();
+# () close = greetingStream.close();
```

Comment:
The variable assignment here is not required, right?

```ballerina
greetingStream.close();
```

should work?

Seems like a bug, created https://github.com/ballerina-platform/ballerina-lang/issues/39000.